assets/javascripts/kitten/components/buttons/button-image-with-text.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react'
+import Radium, { StyleRoot } from 'radium'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import { ButtonImage as ButtonImageBase } from 'kitten/components/buttons/button-image'
+import { Text as TextBase } from 'kitten/components/typography/text'
+import { parseHtml } from 'kitten/helpers/utils/parser'
+
+const ButtonImage = Radium(ButtonImageBase)
+const Text = Radium(TextBase)
+
+class ButtonImageWithTextComponent extends Component {
+  static PropTypes = {
+    title: PropTypes.string.isRequired,
+    description: PropTypes.string.isRequired,
+    isNormal: PropTypes.bool,
+    outsideCard: PropTypes.bool,
+  }
+
+  static defaultProps = {
+    avatarProps: {
+      src: 'https://placehold.it/100x100/caf4fe/caf4fe',
+      alt: '',
+    },
+    title: '',
```

Comment:
On peut se passer de cette ligne. Un attribut qui est `isRequired` n'a pas besoin de `defaultProps` car ca devra dans tous les cas être rempli.

Reply:
J'ai un warning en console, quand je retire les props en `isRequired` des `defaultProps`. 
``Warning: Failed prop type: The prop `description` is marked as required in `ButtonImageWithTextComponent`, but its value is `undefined`.
          in ButtonImageWithTextComponent
``

Reply:
Ca signifie que tu ne mets pas de `description` au niveau de tes exemples. Si c'est `isRequired` il faut que tu ajoutes cette prop.

Reply:
Pourtant si !

Reply:
Ce commentaire est toujours d'actualité ? On peut supprimer `title` et `description` des defaultProps.